stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -196,7 +211,7 @@ internal class DefaultFlowController internal constructor(
         initData: InitData
     ) {
         val confirmParamsFactory = ConfirmParamsFactory(
-            initData.paymentIntent.clientSecret.orEmpty()
+            ClientSecret.PaymentIntentClientSecret(initData.paymentIntent.clientSecret.orEmpty())
```

Comment:
No, it will fail.

Reply:
Should we not allow it to be null then?

Reply:
Sure, but this is a separate change unrelated to this one. I think this is also fine because the error thrown would already explain what's wrong, but I'd need to verify.